<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupUI</class>
 <widget class="QWidget" name="BackupUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>825</width>
    <height>618</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>50000</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_8">
   <item>
    <widget class="QWidget" name="widget_8" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>800</width>
       <height>250</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>1080</width>
       <height>680</height>
      </size>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_9">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QLabel" name="formTitle">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>11</pointsize>
           </font>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>ProMed Backup</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QWidget" name="widget" native="true">
          <property name="maximumSize">
           <size>
            <width>5000</width>
            <height>5000</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">#widget {
	background: #ffffff;
	border-radius:9px;
	border:1px solid #bbbbbb;
}

QWidget {
	background:#fff;
}</string>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <layout class="QVBoxLayout" name="verticalLayout_3">
             <property name="spacing">
              <number>0</number>
             </property>
             <item>
              <spacer name="verticalSpacer_2">
               <property name="orientation">
                <enum>Qt::Vertical</enum>
               </property>
               <property name="sizeType">
                <enum>QSizePolicy::Fixed</enum>
               </property>
               <property name="sizeHint" stdset="0">
                <size>
                 <width>20</width>
                 <height>60</height>
                </size>
               </property>
              </spacer>
             </item>
             <item>
              <widget class="QWidget" name="widget_2" native="true">
               <property name="sizePolicy">
                <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                 <horstretch>0</horstretch>
                 <verstretch>0</verstretch>
                </sizepolicy>
               </property>
               <property name="maximumSize">
                <size>
                 <width>16777215</width>
                 <height>16777215</height>
                </size>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout_2">
                <property name="margin">
                 <number>0</number>
                </property>
                <item>
                 <layout class="QHBoxLayout" name="horizontalLayout">
                  <item>
                   <widget class="QToolButton" name="toolButton">
                    <property name="font">
                     <font>
                      <pointsize>18</pointsize>
                     </font>
                    </property>
                    <property name="styleSheet">
                     <string notr="true">background:none;
border:0px solid #ededed;</string>
                    </property>
                    <property name="text">
                     <string>ProMed Backup</string>
                    </property>
                    <property name="icon">
                     <iconset resource="resources.qrc">
                      <normaloff>:/icons/Backup 96x96</normaloff>:/icons/Backup 96x96</iconset>
                    </property>
                    <property name="iconSize">
                     <size>
                      <width>96</width>
                      <height>96</height>
                     </size>
                    </property>
                    <property name="toolButtonStyle">
                     <enum>Qt::ToolButtonTextUnderIcon</enum>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </widget>
             </item>
             <item>
              <widget class="QWidget" name="widget_3" native="true">
               <layout class="QVBoxLayout" name="verticalLayout_6">
                <item>
                 <layout class="QVBoxLayout" name="verticalLayout_5">
                  <property name="spacing">
                   <number>0</number>
                  </property>
                  <item>
                   <widget class="QLabel" name="label_2">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <property name="font">
                     <font>
                      <pointsize>12</pointsize>
                     </font>
                    </property>
                    <property name="text">
                     <string>To backup your system, click the &quot;Start Backup&quot; button below. Please be patient, this operation might take a long time to complete...</string>
                    </property>
                    <property name="alignment">
                     <set>Qt::AlignCenter</set>
                    </property>
                    <property name="wordWrap">
                     <bool>true</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_6" native="true">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <layout class="QVBoxLayout" name="verticalLayout_9">
                     <property name="topMargin">
                      <number>5</number>
                     </property>
                     <property name="bottomMargin">
                      <number>0</number>
                     </property>
                     <item>
                      <widget class="QWidget" name="widget_7" native="true">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <layout class="QHBoxLayout" name="horizontalLayout_7">
                        <property name="leftMargin">
                         <number>0</number>
                        </property>
                        <property name="topMargin">
                         <number>0</number>
                        </property>
                        <property name="rightMargin">
                         <number>0</number>
                        </property>
                        <property name="bottomMargin">
                         <number>10</number>
                        </property>
                        <item>
                         <layout class="QHBoxLayout" name="horizontalLayout_3">
                          <item>
                           <widget class="QRadioButton" name="radioButton">
                            <property name="font">
                             <font>
                              <pointsize>10</pointsize>
                             </font>
                            </property>
                            <property name="layoutDirection">
                             <enum>Qt::RightToLeft</enum>
                            </property>
                            <property name="text">
                             <string>Online Backup</string>
                            </property>
                           </widget>
                          </item>
                          <item>
                           <widget class="QRadioButton" name="radioButton_2">
                            <property name="font">
                             <font>
                              <pointsize>10</pointsize>
                             </font>
                            </property>
                            <property name="text">
                             <string>Local Backup</string>
                            </property>
                            <property name="checked">
                             <bool>true</bool>
                            </property>
                           </widget>
                          </item>
                         </layout>
                        </item>
                       </layout>
                      </widget>
                     </item>
                     <item>
                      <layout class="QVBoxLayout" name="verticalLayout_8">
                       <item>
                        <widget class="QLabel" name="bak_label">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="font">
                          <font>
                           <pointsize>11</pointsize>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true">color: #666666;</string>
                         </property>
                         <property name="text">
                          <string>(0%) preparing data - creating patients file...</string>
                         </property>
                         <property name="alignment">
                          <set>Qt::AlignCenter</set>
                         </property>
                         <property name="wordWrap">
                          <bool>true</bool>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                  <item>
                   <widget class="QWidget" name="widget_5" native="true">
                    <property name="sizePolicy">
                     <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                      <horstretch>0</horstretch>
                      <verstretch>0</verstretch>
                     </sizepolicy>
                    </property>
                    <layout class="QHBoxLayout" name="horizontalLayout_6">
                     <item>
                      <layout class="QHBoxLayout" name="horizontalLayout_5">
                       <item>
                        <widget class="QProgressBar" name="bak_progress">
                         <property name="sizePolicy">
                          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                           <horstretch>0</horstretch>
                           <verstretch>0</verstretch>
                          </sizepolicy>
                         </property>
                         <property name="minimumSize">
                          <size>
                           <width>300</width>
                           <height>0</height>
                          </size>
                         </property>
                         <property name="font">
                          <font>
                           <pointsize>8</pointsize>
                          </font>
                         </property>
                         <property name="styleSheet">
                          <string notr="true"/>
                         </property>
                         <property name="maximum">
                          <number>100</number>
                         </property>
                         <property name="value">
                          <number>2</number>
                         </property>
                         <property name="textVisible">
                          <bool>true</bool>
                         </property>
                         <property name="invertedAppearance">
                          <bool>false</bool>
                         </property>
                         <property name="textDirection">
                          <enum>QProgressBar::TopToBottom</enum>
                         </property>
                        </widget>
                       </item>
                      </layout>
                     </item>
                    </layout>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QWidget" name="widget_4" native="true">
                  <property name="sizePolicy">
                   <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
                    <horstretch>0</horstretch>
                    <verstretch>0</verstretch>
                   </sizepolicy>
                  </property>
                  <layout class="QVBoxLayout" name="verticalLayout_7">
                   <property name="topMargin">
                    <number>0</number>
                   </property>
                   <item>
                    <layout class="QHBoxLayout" name="horizontalLayout_4">
                     <item>
                      <widget class="QPushButton" name="pushButton">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="font">
                        <font>
                         <pointsize>13</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
	background:#fff;
	background-image:none;
	border-radius: 9px;
	border:1px solid #b1b1b1;
	padding:5px;
	padding-left:10px;
	padding-right:10px;
}

QPushButton:pressed {
	background-color: #555555;
	color: #ffffff;
}</string>
                       </property>
                       <property name="text">
                        <string>Start Backup!</string>
                       </property>
                       <property name="icon">
                        <iconset resource="resources.qrc">
                         <normaloff>:/icons/Save 16x16</normaloff>:/icons/Save 16x16</iconset>
                       </property>
                      </widget>
                     </item>
                     <item>
                      <widget class="QPushButton" name="pushButton_2">
                       <property name="sizePolicy">
                        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                         <horstretch>0</horstretch>
                         <verstretch>0</verstretch>
                        </sizepolicy>
                       </property>
                       <property name="font">
                        <font>
                         <pointsize>13</pointsize>
                        </font>
                       </property>
                       <property name="styleSheet">
                        <string notr="true">QPushButton {
	background:#fff;
	background-image:none;
	border-radius: 9px;
	border:1px solid #b1b1b1;
	padding:5px;
	padding-left:10px;
	padding-right:10px;
}

QPushButton:pressed {
	background-color: #555555;
	color: #ffffff;
}</string>
                       </property>
                       <property name="text">
                        <string>Restore From Backup</string>
                       </property>
                       <property name="icon">
                        <iconset resource="resources.qrc">
                         <normaloff>:/icons/Import 24x24</normaloff>:/icons/Import 24x24</iconset>
                       </property>
                      </widget>
                     </item>
                    </layout>
                   </item>
                  </layout>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
